package.json

```diff
@@ -7,10 +7,11 @@
   "scripts": {
     "test": "node test/index.js",
     "build:html": "npx @11ty/eleventy --config=_build/.eleventy.cjs",
-    "build": "npm run build:html",
+    "build": "npm run build:html && npm run build:js",
     "watch:html": "npx @11ty/eleventy --config=_build/.eleventy.cjs --watch",
     "watch": "npm run watch:html",
-    "build:docs": "npx typedoc --out docs src/"
+    "build:docs": "npx typedoc --out docs src/",
+    "build:js": "rollup -c"
```

Comment:
Wait, why *delete* `npx`? That was needed regardless! Please read up a bit on what it does and you'll see why.

Reply:
Sorry, I got confused from the slack conversation